doc/ref_model/chapters/chapter03.md

```diff
@@ -19,7 +19,7 @@
 * [3.8 Hardware Acceleration Abstraction](#3.8)
   * [3.8.1 Types of accelerators](#3.8.1)
   * [3.8.2 Levels of accelerator consumption](#3.8.2)
-  * [3.8.3 Decoupling applications from infrastructure with Hardware Acceleration](#3.8.3)
+  * [3.8.9 Decoupling Applications from Infrastructure and Platform with Hardware Acceleration](#3.8.9)
```

Comment:
What is planned for 3.8.3, ...3.8.8? (Since you renumbered to 3.8.9)

Reply:
> What is planned for 3.8.3, ...3.8.8? (Since you renumbered to 3.8.9)

Please in #2051 scroll down to list "This was referenced x days ago".

Reply:
OK. Was a bit hidden ;-)